### PR TITLE
User read

### LINKED ISSUE #119: [Backend | Fetch User Info] Fetch User Handler
Add ability for end users to fetch info of their own user through the REST API

### PR DESCRIPTION
**Pull request by Jacob4649**
Closes #119. Adds a route for fetching information on a user. Only the currently logged in user can fetch this information. It is possible that in the future more ways to fetch user info will be added that do not only allow the currently logged in user to see this information, but these other versions will likely provide fewer pieces of sensitive information.

### CODE REVIEW

**Review by Enkhzul001 — APPROVED** [review]
looks good to me

**Review by BullDF — APPROVED** [review]
Very helpful method!

**Review by stevolopolis — APPROVED** [review]
Nice namings for Users :)

No comment, very straight forward code.

File: backend/src/test/java/com/javahelp/backend/endpoint/user/ReadHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+ * Tests for {@link ReadHandler}
+ */
+public class ReadHandlerTest {
+
+    IUserStore store = IUserStore.getDefaultImplementation();
+    ITokenStore tokenStore = ITokenStore.getDefaultImplementation();
+
+    private static final String USER_ENDPOINT = "https://gwkvm1k2j5.execute-api.us-east-1.amazonaws.com/users/";
+
+    ClientUserInfo info = new ClientUserInfo("i want to sleep",
```
Comment (Lam207) [inline backend/src/test/java/com/javahelp/backend/endpoint/user/ReadHandlerTest.java:33]: lol you really put this in context. Hopefully you will get some sleep soon this holiday :D

**Review by Lam207 — APPROVED** [review]
Another great work!

**Pull request merged**